### ISSUE TITLE
[BUG] Github release (btop-x86_64-linux-musl) contains additional folders

### ISSUE THREAD

**Issue by saltydk** [aristocratos/btop#465]
Is it intended that the the release tbz (have only checked btop-x86_64-linux-musl) contains additional folders now?

Before
![image](https://user-images.githubusercontent.com/6587950/200402539-1483a1f4-8efa-4e33-9a2f-914039dffd19.png)

After
![image](https://user-images.githubusercontent.com/6587950/200402575-041ee268-ea3e-4b39-86d9-52372a0cd7bc.png)

**Comment by theofficialgman** [comment]
yes I need to know this as well as it breaks automation at pi-apps

**Comment by aristocratos** [comment]
@saltydk @theofficialgman 
See  #242
Forgot to mention in the changelog, but didn't actually think anyone was relying on the release packages for package management.

I'll make sure to give notice beforehand if for any reason the tar structure would change again (unlikely that it will).
